lnwallet/config.go

```diff
@@ -12,4 +17,19 @@ type Config struct {
 	// max htlcs in flight?
 	// possible secret derivation functions
 	//
+
+	// Number of max pending channels which might be created with one
+	// peer. In both cases - locally (RPC client) and remotely
+	// (Node interaction), error will be returned if the number of pending
+	// channels will exceed maximum.
+	MaxPendingChannels int
```

Comment:
Since this a p2p policy configuration parameter, the wallet doesn't need to know of it. Therefore, the changes in this file should be reverted.